src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -266,7 +280,7 @@ private String waitFor() throws InterruptedException {
             Logger.debug(
                 this, "#waitFor(): process finished: %s", this.process
             );
-            if (!done.await(2L, TimeUnit.SECONDS)) {
```

Comment:
So we are waiting for each monitor for 1 second? Why 1 second per monitor? Previously we had only one monitor and we waited 2 seconds also.